qstack/spahm/rho/bond.py

```diff
@@ -78,6 +79,13 @@ def main():
     if args.print>0: print(vars(args))
     correct_num_threads()
 
+    import pyscf.data.elements as pysymb
```

Comment:
* I don't think it's good to forcely set ecp. Better raise an error
* This checking code should be a separate function in `compound.py`, if not inside of `xyz_to_mol`